homeassistant/helpers/service_info/bluetooth.py

```diff
@@ -1,4 +1,6 @@
 """The bluetooth integration service info."""
-from home_assistant_bluetooth import (  # pylint: disable=unused-import # noqa: F401
+from home_assistant_bluetooth import (  # pylint: disable=unused-import
```

Comment:
Doesn't pylint recognize that we define `__all__` below with this name?

Reply:
Pylint does recognize it. I removed it.